h2o-py/h2o/model/extensions/fairness_metrics.py

```diff
@@ -0,0 +1,35 @@
+import h2o
+from h2o.utils.typechecks import assert_is_type
+
+
+class FairnessMetrics:
```

Comment:
gonna be annoying here, but can't we rename this to just `Fairness`?
An extension is allowed to have multiple methods, and I guess that if we can apply fairness-related functionalities to a model, it's likely to support them all (when specific to a certain type of model, we can always extend the class/mixin anyway as we do... with GLM).
Also, we use the `Metrics` suffixes for many other classes not related with this one, so can be confusing at this one works very differently.

For example, shouldn't the fairness plot functions applicable to a single model also be aliased here?
I'm ok with having more powerful functional-style of functions in `h2o.explain` module (although I'm not sure if everything should go under that specific module, maybe need submodules?, as it's turning into a very generic plot module with no structure according to functionality), but Python is at least as much (if not more) OO than it is functional, and many users may prefer the OO approach, especially to easily discover what can be done with their model.

Reply:
> gonna be annoying here, but can't we rename this to just Fairness?
The only annoying thing here is that I didn't realize I could use these mixins instead of the explain module. I think refactoring the explain module itself into multiple mixins could simplify the logic significantly.  (The python explain module was heavily influenced by R's and at that time we didn't use the extensions (the mixins) in python.)

Reply:
@ledell I would refactor it in future PR if there will be time (since it shouldn't affect the user; it would be just nicer for us).